app/views/sequences/show.html.erb

```diff
@@ -85,13 +87,33 @@
                   </td>
                   <td><%= entry.name %></td>
                   <td><%= link_to entry.taxon.name, "http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id="+entry.taxon.id.to_s, :target => "_blank" %></td>
-                  <td><%= raw(entry.ec_cross_references.map{|ec| "<a href='http://enzyme.expasy.org/EC/"+ec.ec_id+"' target='_blank'>"+ec.ec_id+"</a>"}.join(", ")) %></td>
+                  <td><%= raw(entry.ec_cross_references.map{|ec| "<a href='http://enzyme.expasy.org/EC/"+ec.ec_number+"' target='_blank'>"+ec.ec_number+"</a>"}.join(", ")) %></td>
                   <td><%= raw(entry.go_cross_references.map{|go| "<a href='http://amigo.geneontology.org/amigo/term/"+go.go_id+"' target='_blank'>"+go.go_id+"</a>"}.join(", ")) %></td>
                 </tr>
               <% end %>
             <% end %>
           </table>
         </div>
+        <div class="tab-pane" id="ec-table">
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+                <% for column in @column_name do%>
+                  <th><%= column %></th>
+                <% end %>
+              </tr>
+            </thead>
+            <% for ec_key, ec_value in @ec_lca_table do%>
+                <td><strong><%= ec_key %></strong></td>
```

Comment:
These should be wrapped in a row tag.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3386) by @bmesuere on Wed Oct 21 2015 at 16:51._